Finish mobile version

Praveen1995-10-14 opened the pull request:
This is a mobile version of Portfolio.

I added an HTML file with the content.
I added CSS to design the HTML content as per requirements.
I added a .gitignore file to filter the unwanted files.
I added the linters to solve the error in HTML and CSS files.

### **### This is my previous pull request https://github.com/Praveen1995-10-14/myportfolio/pull/4**
### I never knew I shouldn't created a new pull request.

jerryowusu commented on style.css:
```diff
@@ -138,3 +163,539 @@ ul {
+  margin-top: 4px;
+}
+
+.seeProj {
+  width: 162px;
+  height: 48px;
+  margin-top: 12px;
+  border-radius: 8px;
+  padding-left: -12px;
+  text-align: center;
+}
```
As the previous reviewer said, Please kindly implement the active, pressed and hover effects of all the buttons. It is true that these effects are not seen on mobile versions but it is a must to implement them at this stage of the project. (_check the last project bulleted point in the image below_).You can check the desktop side of the figma design for the button effects. :pray:

![Screenshot from 2022-09-23 16-24-44](https://user-images.githubusercontent.com/65233860/192027091-73092812-c1e4-4bca-a444-fddd658be04c.png)

Praveen1995-10-14 replied:
hover is not mandatory 
actually i put it by self but when reviewer told me use hover here so I removed that feature

Praveen1995-10-14 replied:
### I used ul li tag for html,css and js for this section
and earlier reviewer told me to use h tag in work_section bcz earlier I was using div.
i changed everything which she said 
except hover bcz that is not so compulsory.
few hours left for deadline,plz don't block for that reason.

jerryowusu commented on index.html:
```diff
@@ -39,6 +40,221 @@ <h3>LET'S CONNECT</h3>
                 </section>
         </div>
+         <!-- work -->
+         <div id="work_1">
+            <div class="project">
+                <img src="images/header/tonic.png" alt="tonic">
+            </div>
+            <h4 class="project_detail">Tonic</h4>
+            <div class="job_detail">
+                <h4 class="canopy">CANOPY</h4>
+                <p class="dot">•</p>
+                <p class="job">Back End Dev</p>
+                <p class="dot">•</p>
+                <p class="yrs">2015</p>
+            </div>
+            <div class="read">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="progm_skills">
+                <ul>
+                    <li class="skill_list1">
+                        <button>html</button>
+                    </li>
+                    <li class="skill_list2">
+                        <button>CSS</button>
+                    </li>
+                    <li class="skill_list3">
+                        <button>javaScript</button>
```
[OPTIONAL] As stated in the project requirement below, you are supposed to create two sections at these stage of the project, one section for the `works/projects` and another section for the `about me`. 
 
![Screenshot from 2022-09-24 12-11-24](https://user-images.githubusercontent.com/65233860/192097230-5503370a-0c23-4a88-8e25-599798091143.png)

Praveen1995-10-14 replied:
### I used ul li tag for html,css and js for this section
and earlier reviewer told me to use h tag in work_section bcz earlier I was using div.
i changed everything which she said 
except hover bcz that is not so compulsory.
few hours left for deadline,plz don't block for that reason.

jerryowusu commented on index.html:
```diff
@@ -39,6 +40,221 @@ <h3>LET'S CONNECT</h3>
+            <div class="read">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="progm_skills">
+                <ul>
+                    <li class="skill_list1">
+                        <button>html</button>
+                    </li>
+                    <li class="skill_list2">
+                        <button>CSS</button>
+                    </li>
+                    <li class="skill_list3">
+                        <button>javaScript</button>
+                    </li>
```
[OPTIONAL] These are list items and not buttons, so using ul and li tags to list them is ok. No need for button tags since they are not buttons

jerryowusu commented on index.html:
```diff
@@ -39,6 +40,221 @@ <h3>LET'S CONNECT</h3>
+                <img src="images/header/second.png" alt="tonic">
+            </div>
+            <h4 class="project_detail">Multi-Post Stories</h4>
+            <div class="job_detail">
+                <h4 class="canopy">CANOPY</h4>
+                <p class="dot">•</p>
+                <p class="job">Back End Dev</p>
+                <p class="dot">•</p>
+                <p class="yrs">2015</p>
+            </div>
```
[OPTIONAL] Kindly use ul and li tags to list such items as these and not p tags. The will help your code be more presentable, clean and conventional for best practices.

jerryowusu reviewed (changes requested):
Hi @Praveen1995-10-14 :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![so close](https://user-images.githubusercontent.com/65233860/192125697-78f5563a-a79f-472d-af73-461d1894a789.gif)

 ## Highlights
:heavy_check_mark: All linters are passing
:heavy_check_mark: Good commit messages
:heavy_check_mark: Followed GitHub flow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Praveen1995-10-14 commented:
*hover functions are not used in Figma I used that feature by myself only,
so when the reviewer saw my shower then she told me to use hover here.*
*then I removed that feature.*

On Sat, Sep 24, 2022 at 8:34 PM Jerry Owusu ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @Praveen1995-10-14 <https://github.com/Praveen1995-10-14> 👋
>
> Great work on making the changes requested by a previous reviewer 👏
> You've done well implementing some of the requested changes, but there are
> still some which aren't addressed yet.
>
> [image: so close]
> <https://user-images.githubusercontent.com/65233860/192125697-78f5563a-a79f-472d-af73-461d1894a789.gif>
> Highlights
>
> ✔️ All linters are passing
> ✔️ Good commit messages
> ✔️ Followed GitHub flow
> Required Changes ♻️
>
> *Check the comments under the review.*
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix won't stop the approval of this
> PR. However, I strongly recommend you to take them into account as they can
> make your code better. Some of them were simply missed by the previous
> reviewer and addressing them will really improve your application.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In style.css
> <https://github.com/Praveen1995-10-14/myportfolio/pull/5#discussion_r979342916>
> :
>
> > +.seeProj {
>
> +  width: 162px;
>
> +  height: 48px;
>
> +  margin-top: 12px;
>
> +  border-radius: 8px;
>
> +  padding-left: -12px;
>
> +  text-align: center;
>
> +}
>
>
> As the previous reviewer said, Please kindly implement the active, pressed
> and hover effects of all the buttons. It is true that these effects are not
> seen on mobile versions but it is a must to implement them at this stage of
> the project. (*check the last project bulleted point in the image below*).You
> can check the desktop side of the figma design for the button effects. 🙏
>
> [image: Screenshot from 2022-09-23 16-24-44]
> <https://user-images.githubusercontent.com/65233860/192027091-73092812-c1e4-4bca-a444-fddd658be04c.png>
> ------------------------------
>
> In index.html
> <https://github.com/Praveen1995-10-14/myportfolio/pull/5#discussion_r979343305>
> :
>
> > +         <div id="work_1">
>
> +            <div class="project">
>
> +                <img src="images/header/tonic.png" alt="tonic">
>
> +            </div>
>
> +            <h4 class="project_detail">Tonic</h4>
>
> +            <div class="job_detail">
>
> +                <h4 class="canopy">CANOPY</h4>
>
> +                <p class="dot">•</p>
>
> +                <p class="job">Back End Dev</p>
>
> +                <p class="dot">•</p>
>
> +                <p class="yrs">2015</p>
>
> +            </div>
>
> +            <div class="read">
>
> +                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
>
> +            </div>
>
> +            <div class="progm_skills">
>
> +                <ul>
>
> +                    <li class="skill_list1">
>
> +                        <button>html</button>
>
> +                    </li>
>
> +                    <li class="skill_list2">
>
> +                        <button>CSS</button>
>
> +                    </li>
>
> +                    <li class="skill_list3">
>
> +                        <button>javaScript</button>
>
>
> [OPTIONAL] As stated in the project requirement below, you are supposed to
> create two sections at these stage of the project, one section for the
> works/projects and another section for the about me.
>
> [image: Screenshot from 2022-09-24 12-11-24]
> <https://user-images.githubusercontent.com/65233860/192097230-5503370a-0c23-4a88-8e25-599798091143.png>
> ------------------------------
>
> In index.html
> <https://github.com/Praveen1995-10-14/myportfolio/pull/5#discussion_r979343631>
> :
>
> > +            <div class="progm_skills">
>
> +                <ul>
>
> +                    <li class="skill_list1">
>
> +                        <button>html</button>
>
> +                    </li>
>
> +                    <li class="skill_list2">
>
> +                        <button>CSS</button>
>
> +                    </li>
>
> +                    <li class="skill_list3">
>
> +                        <button>javaScript</button>
>
> +                    </li>
>
>
> [OPTIONAL] These are list items and not buttons, so using ul and li tags
> to list them is ok. No need for button tags since they are not buttons
> ------------------------------
>
> In index.html
> <https://github.com/Praveen1995-10-14/myportfolio/pull/5#discussion_r979343760>
> :
>
> > +            <div class="job_detail">
>
> +                <h4 class="canopy">CANOPY</h4>
>
> +                <p class="dot">•</p>
>
> +                <p class="job">Back End Dev</p>
>
> +                <p class="dot">•</p>
>
> +                <p class="yrs">2015</p>
>
> +            </div>
>
>
> [OPTIONAL] Kindly use ul and li tags to list such items as these and not p
> tags. The will help your code be more presentable, clean and conventional
> for best practices.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Praveen1995-10-14/myportfolio/pull/5#pullrequestreview-1119362749>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATKNODFWE3MPS5OHR66OXRLV7623FANCNFSM6AAAAAAQU2K5JM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Praveen1995-10-14 commented:
*What specific changes do I have to do bcz many things are optional and I
have the last review left*
plz approved my project bcz the hover function is not compulsory and for
that plz don't disapprove my project.

On Sat, Sep 24, 2022 at 8:50 PM praveen singh ***@***.***> wrote:

> *hover functions are not used in Figma I used that feature by myself only,
> so when the reviewer saw my shower then she told me to use hover here.*
> *then I removed that feature.*
>
> On Sat, Sep 24, 2022 at 8:34 PM Jerry Owusu ***@***.***>
> wrote:
>
>> ***@***.**** requested changes on this pull request.
>>
>> Hi @Praveen1995-10-14 <https://github.com/Praveen1995-10-14> 👋
>>
>> Great work on making the changes requested by a previous reviewer 👏
>> You've done well implementing some of the requested changes, but there
>> are still some which aren't addressed yet.
>>
>> [image: so close]
>> <https://user-images.githubusercontent.com/65233860/192125697-78f5563a-a79f-472d-af73-461d1894a789.gif>
>> Highlights
>>
>> ✔️ All linters are passing
>> ✔️ Good commit messages
>> ✔️ Followed GitHub flow
>> Required Changes ♻️
>>
>> *Check the comments under the review.*
>> Optional suggestions
>>
>> *Every comment with the [OPTIONAL] prefix won't stop the approval of this
>> PR. However, I strongly recommend you to take them into account as they can
>> make your code better. Some of them were simply missed by the previous
>> reviewer and addressing them will really improve your application.*
>>
>> Cheers and Happy coding!👏👏👏
>>
>> Feel free to leave any questions or comments in the PR thread if
>> something is not 100% clear.
>> Please, remember to tag me in your question so I can receive the
>> notification.
>>
>> *Please, do not open a new Pull Request for re-reviews. You should use
>> the same Pull Request submitted for the first review, either valid or
>> invalid unless it is requested otherwise.*
>> ------------------------------
>>
>> *As described in the Code reviews limits policy
>> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
>> limited number of reviews per project (check the exact number in your
>> Dashboard). If you think that the code review was not fair, you can request
>> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>> ------------------------------
>>
>> In style.css
>> <https://github.com/Praveen1995-10-14/myportfolio/pull/5#discussion_r979342916>
>> :
>>
>> > +.seeProj {
>>
>> +  width: 162px;
>>
>> +  height: 48px;
>>
>> +  margin-top: 12px;
>>
>> +  border-radius: 8px;
>>
>> +  padding-left: -12px;
>>
>> +  text-align: center;
>>
>> +}
>>
>>
>> As the previous reviewer said, Please kindly implement the active,
>> pressed and hover effects of all the buttons. It is true that these effects
>> are not seen on mobile versions but it is a must to implement them at this
>> stage of the project. (*check the last project bulleted point in the
>> image below*).You can check the desktop side of the figma design for the
>> button effects. 🙏
>>
>> [image: Screenshot from 2022-09-23 16-24-44]
>> <https://user-images.githubusercontent.com/65233860/192027091-73092812-c1e4-4bca-a444-fddd658be04c.png>
>> ------------------------------
>>
>> In index.html
>> <https://github.com/Praveen1995-10-14/myportfolio/pull/5#discussion_r979343305>
>> :
>>
>> > +         <div id="work_1">
>>
>> +            <div class="project">
>>
>> +                <img src="images/header/tonic.png" alt="tonic">
>>
>> +            </div>
>>
>> +            <h4 class="project_detail">Tonic</h4>
>>
>> +            <div class="job_detail">
>>
>> +                <h4 class="canopy">CANOPY</h4>
>>
>> +                <p class="dot">•</p>
>>
>> +                <p class="job">Back End Dev</p>
>>
>> +                <p class="dot">•</p>
>>
>> +                <p class="yrs">2015</p>
>>
>> +            </div>
>>
>> +            <div class="read">
>>
>> +                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
>>
>> +            </div>
>>
>> +            <div class="progm_skills">
>>
>> +                <ul>
>>
>> +                    <li class="skill_list1">
>>
>> +                        <button>html</button>
>>
>> +                    </li>
>>
>> +                    <li class="skill_list2">
>>
>> +                        <button>CSS</button>
>>
>> +                    </li>
>>
>> +                    <li class="skill_list3">
>>
>> +                        <button>javaScript</button>
>>
>>
>> [OPTIONAL] As stated in the project requirement below, you are supposed
>> to create two sections at these stage of the project, one section for the
>> works/projects and another section for the about me.
>>
>> [image: Screenshot from 2022-09-24 12-11-24]
>> <https://user-images.githubusercontent.com/65233860/192097230-5503370a-0c23-4a88-8e25-599798091143.png>
>> ------------------------------
>>
>> In index.html
>> <https://github.com/Praveen1995-10-14/myportfolio/pull/5#discussion_r979343631>
>> :
>>
>> > +            <div class="progm_skills">
>>
>> +                <ul>
>>
>> +                    <li class="skill_list1">
>>
>> +                        <button>html</button>
>>
>> +                    </li>
>>
>> +                    <li class="skill_list2">
>>
>> +                        <button>CSS</button>
>>
>> +                    </li>
>>
>> +                    <li class="skill_list3">
>>
>> +                        <button>javaScript</button>
>>
>> +                    </li>
>>
>>
>> [OPTIONAL] These are list items and not buttons, so using ul and li tags
>> to list them is ok. No need for button tags since they are not buttons
>> ------------------------------
>>
>> In index.html
>> <https://github.com/Praveen1995-10-14/myportfolio/pull/5#discussion_r979343760>
>> :
>>
>> > +            <div class="job_detail">
>>
>> +                <h4 class="canopy">CANOPY</h4>
>>
>> +                <p class="dot">•</p>
>>
>> +                <p class="job">Back End Dev</p>
>>
>> +                <p class="dot">•</p>
>>
>> +                <p class="yrs">2015</p>
>>
>> +            </div>
>>
>>
>> [OPTIONAL] Kindly use ul and li tags to list such items as these and not
>> p tags. The will help your code be more presentable, clean and conventional
>> for best practices.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Praveen1995-10-14/myportfolio/pull/5#pullrequestreview-1119362749>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ATKNODFWE3MPS5OHR66OXRLV7623FANCNFSM6AAAAAAQU2K5JM>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

Praveen1995-10-14 commented:
*I used ul li tag for html,css and js for this sectionand earlier reviewer
told me to use h tag in work_section bcz earlier I was using div.*


*i changed everything which she said except hover bcz that is not so
compulsory.few hours left for deadline,plz don't block for that reason.*

On Sat, Sep 24, 2022 at 8:59 PM praveen singh ***@***.***> wrote:

> *What specific changes do I have to do bcz many things are optional and I
> have the last review left*
> plz approved my project bcz the hover function is not compulsory and for
> that plz don't disapprove my project.
>
> On Sat, Sep 24, 2022 at 8:50 PM praveen singh ***@***.***> wrote:
>
>> *hover functions are not used in Figma I used that feature by myself
>> only, so when the reviewer saw my shower then she told me to use hover
>> here.*
>> *then I removed that feature.*
>>
>> On Sat, Sep 24, 2022 at 8:34 PM Jerry Owusu ***@***.***>
>> wrote:
>>
>>> ***@***.**** requested changes on this pull request.
>>>
>>> Hi @Praveen1995-10-14 <https://github.com/Praveen1995-10-14> 👋
>>>
>>> Great work on making the changes requested by a previous reviewer 👏
>>> You've done well implementing some of the requested changes, but there
>>> are still some which aren't addressed yet.
>>>
>>> [image: so close]
>>> <https://user-images.githubusercontent.com/65233860/192125697-78f5563a-a79f-472d-af73-461d1894a789.gif>
>>> Highlights
>>>
>>> ✔️ All linters are passing
>>> ✔️ Good commit messages
>>> ✔️ Followed GitHub flow
>>> Required Changes ♻️
>>>
>>> *Check the comments under the review.*
>>> Optional suggestions
>>>
>>> *Every comment with the [OPTIONAL] prefix won't stop the approval of
>>> this PR. However, I strongly recommend you to take them into account as
>>> they can make your code better. Some of them were simply missed by the
>>> previous reviewer and addressing them will really improve your application.*
>>>
>>> Cheers and Happy coding!👏👏👏
>>>
>>> Feel free to leave any questions or comments in the PR thread if
>>> something is not 100% clear.
>>> Please, remember to tag me in your question so I can receive the
>>> notification.
>>>
>>> *Please, do not open a new Pull Request for re-reviews. You should use
>>> the same Pull Request submitted for the first review, either valid or
>>> invalid unless it is requested otherwise.*
>>> ------------------------------
>>>
>>> *As described in the Code reviews limits policy
>>> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
>>> limited number of reviews per project (check the exact number in your
>>> Dashboard). If you think that the code review was not fair, you can request
>>> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>>> ------------------------------
>>>
>>> In style.css
>>> <https://github.com/Praveen1995-10-14/myportfolio/pull/5#discussion_r979342916>
>>> :
>>>
>>> > +.seeProj {
>>>
>>> +  width: 162px;
>>>
>>> +  height: 48px;
>>>
>>> +  margin-top: 12px;
>>>
>>> +  border-radius: 8px;
>>>
>>> +  padding-left: -12px;
>>>
>>> +  text-align: center;
>>>
>>> +}
>>>
>>>
>>> As the previous reviewer said, Please kindly implement the active,
>>> pressed and hover effects of all the buttons. It is true that these effects
>>> are not seen on mobile versions but it is a must to implement them at this
>>> stage of the project. (*check the last project bulleted point in the
>>> image below*).You can check the desktop side of the figma design for
>>> the button effects. 🙏
>>>
>>> [image: Screenshot from 2022-09-23 16-24-44]
>>> <https://user-images.githubusercontent.com/65233860/192027091-73092812-c1e4-4bca-a444-fddd658be04c.png>
>>> ------------------------------
>>>
>>> In index.html
>>> <https://github.com/Praveen1995-10-14/myportfolio/pull/5#discussion_r979343305>
>>> :
>>>
>>> > +         <div id="work_1">
>>>
>>> +            <div class="project">
>>>
>>> +                <img src="images/header/tonic.png" alt="tonic">
>>>
>>> +            </div>
>>>
>>> +            <h4 class="project_detail">Tonic</h4>
>>>
>>> +            <div class="job_detail">
>>>
>>> +                <h4 class="canopy">CANOPY</h4>
>>>
>>> +                <p class="dot">•</p>
>>>
>>> +                <p class="job">Back End Dev</p>
>>>
>>> +                <p class="dot">•</p>
>>>
>>> +                <p class="yrs">2015</p>
>>>
>>> +            </div>
>>>
>>> +            <div class="read">
>>>
>>> +                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
>>>
>>> +            </div>
>>>
>>> +            <div class="progm_skills">
>>>
>>> +                <ul>
>>>
>>> +                    <li class="skill_list1">
>>>
>>> +                        <button>html</button>
>>>
>>> +                    </li>
>>>
>>> +                    <li class="skill_list2">
>>>
>>> +                        <button>CSS</button>
>>>
>>> +                    </li>
>>>
>>> +                    <li class="skill_list3">
>>>
>>> +                        <button>javaScript</button>
>>>
>>>
>>> [OPTIONAL] As stated in the project requirement below, you are supposed
>>> to create two sections at these stage of the project, one section for the
>>> works/projects and another section for the about me.
>>>
>>> [image: Screenshot from 2022-09-24 12-11-24]
>>> <https://user-images.githubusercontent.com/65233860/192097230-5503370a-0c23-4a88-8e25-599798091143.png>
>>> ------------------------------
>>>
>>> In index.html
>>> <https://github.com/Praveen1995-10-14/myportfolio/pull/5#discussion_r979343631>
>>> :
>>>
>>> > +            <div class="progm_skills">
>>>
>>> +                <ul>
>>>
>>> +                    <li class="skill_list1">
>>>
>>> +                        <button>html</button>
>>>
>>> +                    </li>
>>>
>>> +                    <li class="skill_list2">
>>>
>>> +                        <button>CSS</button>
>>>
>>> +                    </li>
>>>
>>> +                    <li class="skill_list3">
>>>
>>> +                        <button>javaScript</button>
>>>
>>> +                    </li>
>>>
>>>
>>> [OPTIONAL] These are list items and not buttons, so using ul and li tags
>>> to list them is ok. No need for button tags since they are not buttons
>>> ------------------------------
>>>
>>> In index.html
>>> <https://github.com/Praveen1995-10-14/myportfolio/pull/5#discussion_r979343760>
>>> :
>>>
>>> > +            <div class="job_detail">
>>>
>>> +                <h4 class="canopy">CANOPY</h4>
>>>
>>> +                <p class="dot">•</p>
>>>
>>> +                <p class="job">Back End Dev</p>
>>>
>>> +                <p class="dot">•</p>
>>>
>>> +                <p class="yrs">2015</p>
>>>
>>> +            </div>
>>>
>>>
>>> [OPTIONAL] Kindly use ul and li tags to list such items as these and not
>>> p tags. The will help your code be more presentable, clean and conventional
>>> for best practices.
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/Praveen1995-10-14/myportfolio/pull/5#pullrequestreview-1119362749>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/ATKNODFWE3MPS5OHR66OXRLV7623FANCNFSM6AAAAAAQU2K5JM>
>>> .
>>> You are receiving this because you were mentioned.Message ID:
>>> ***@***.***>
>>>
>>

jerryowusu commented:
Hello @Praveen1995-10-14, I know this can be very frustrating and I am sorry for any inconvenience caused. For now the only thing left for the approval of your project is the the one without the `[OPTIONAL]` tag ie: the implementation of the button interactions (hover, enabled, pressed, etc). It is a requirement at this stage since it was stated in your project requirement. You can find these button effects on the desktop side of the figma design, check the image below.

jerryowusu commented:
Hello @Praveen1995-10-14, you don't have to assign me or any other person to review. Kindly use the dashboard to submit for review just as you did before.

emmiiorji reviewed (approved):
Hi @Praveen1995-10-14, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![congratulations](https://user-images.githubusercontent.com/20554583/192127678-1419abb3-531d-4f6a-920d-5efdfea8a556.gif)


## Good Highlights✔

- Good commit messages
- Implemented requested changes
- Adherence to GitHub flow

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAMMAS-SALEEM reviewed (commented):
# Status: Approved :green_circle: :1st_place_medal: :100:
![success](https://camo.githubusercontent.com/6ada59f6189497c1c2cae6d06f8154b7307a58643f232340e288140ed469eb28/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966)

## Hey @Praveen1995-10-14 :wave:

Great work on implementing the requirements. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

:heavy_check_mark: Following Requirements
:heavy_check_mark: Descriptive PR Title

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
